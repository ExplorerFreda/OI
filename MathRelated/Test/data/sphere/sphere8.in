10
19649.000000 3020.500000 2411.000000 18895.000000 18121.000000 12507.500000 12726.000000 19390.500000 1491.000000 14048.500000
1633.000000 9449.500000 2186.500000 14356.500000 2827.500000 6749.000000 18081.000000 19330.000000 15797.000000 18867.000000
1096.000000 8529.000000 1033.500000 1384.000000 6453.500000 8302.500000 1566.000000 10356.500000 17439.500000 11623.500000
6783.000000 1655.500000 17520.500000 3205.500000 18943.500000 19269.500000 5517.500000 19382.000000 2028.000000 1341.500000
14215.000000 19887.000000 10701.500000 18634.500000 13683.500000 17148.500000 2389.500000 13770.000000 11101.500000 2167.000000
19294.000000 3811.000000 11184.000000 18207.000000 13939.000000 7623.000000 8932.000000 6563.500000 9077.000000 16198.500000
17590.500000 12469.000000 3883.000000 12162.500000 17395.000000 18096.000000 4841.000000 6785.000000 18593.000000 12622.500000
11429.000000 13410.000000 19001.000000 13269.000000 16028.500000 19832.000000 3579.000000 5700.500000 18042.000000 12114.500000
16966.000000 11772.000000 17851.000000 11269.000000 7323.000000 15048.000000 13095.000000 12676.000000 14775.500000 18956.000000
17908.000000 12657.500000 15268.000000 461.500000 7080.500000 1153.500000 11157.500000 875.500000 2063.000000 341.500000
8133.000000 6854.000000 1230.000000 8485.000000 13511.500000 1625.500000 11996.000000 3517.500000 13226.500000 3355.500000
