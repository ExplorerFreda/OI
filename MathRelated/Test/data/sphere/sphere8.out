-5220.136 6558.694 -4347.860 16069.742 -12773.569 48088.796 20217.014 -17870.115 -21102.147 10564.474
